validations/src/validations.rs

```diff
@@ -943,6 +948,24 @@ fn create_expected_tally_transaction(
     Ok((ta_tx, dr_state.clone()))
 }
 
+pub fn calculate_errors_and_liars_count_from_tally(
```

Comment:
This funcion is named `calculate_errors_and_liars` but it returns `(liars, errors)`